libd/src/d/semantic/symbol.d

```diff
@@ -732,11 +743,81 @@ struct SymbolAnalyzer {
 			
 			baseMethods = baseMethods[0 .. i];
 		}
-		
+
 		c.members = cast(Symbol[]) baseFields;
 		c.members ~= baseMethods;
 		scheduler.require(members);
 		c.members ~= members;
+
+		bool found = false;
+		foreach (i; c.interfaces){
+			scheduler.require(i);
+
+			auto vtblType = Type.get(BuiltinType.Void).getPointer(TypeQualifier.Immutable);
+			
+			// XXX: d is hijacked without explicit import
+			import d.context.name : BuiltinName;
+			
+			// TODO: use defaultinit.
+			auto ivtbl = new Field(d.location, 0, vtblType, BuiltinName!"__ivtbl", null);
+			c.members ~= ivtbl;
```

Comment:
No. The interface vtbl is not a field. There is only one vtbl field. The interface vtbl it part of the class vtbl.